tests/PySys/call_tedge_-h_-V/run.py

```diff
@@ -0,0 +1,30 @@
+import pysys
+from pysys.basetest import BaseTest
+
+import time
+
+"""
+Validate command line option -h -V
+
+Given a running system
+When we call tedge -h -V
+Then we find the string USAGE: in the output of stderr
+
+Note: This should probably return a non zero exit status and complain
```

Comment:
run `tedge -h -V` returns `0` in my Ubuntu18.04.

Reply:
Absolutely, that's why I wrote a bugreport: CIT-318 :)

Reply:
This test is disabled as well. Sorry, that's hard to see in this PR

Reply:
No problem. I got it.